pkg/snapshot/snapshotter/garbagecollector.go

```diff
@@ -193,24 +192,28 @@ func getSnapStreamIndexList(snapList brtypes.SnapList) []int {
 	return snapStreamIndexList
 }
 
-// garbageCollectChunks deletes the chunks in the store from snaplist starting at index low (inclusive) till high (exclusive).
-func garbageCollectChunks(store brtypes.SnapStore, snapList brtypes.SnapList, low, high int) {
-	for index := low; index < high; index++ {
+// garbageCollectChunks deletes the chunks in the store from snaplist.
```

Comment:
Shall we improvise the comment something like below
WDYT ? 
```suggestion
// garbageCollectChunks goes through the snapList and deletes any chunks.
// It returns a list of non-chunk snapshots, preserving the original order.
```